packages/protocol-kit/src/contracts/SafeProxyFactory/v1.0.0/SafeProxyFactoryContract_v1_0_0.ts

```diff
@@ -38,7 +40,7 @@ class SafeProxyFactoryContract_v1_0_0
     safeProvider: SafeProvider,
     customContractAddress?: string,
     customContractAbi?: SafeProxyFactoryContract_v1_0_0_Abi,
-    runner?: ContractRunner | null
+    runner?: PublicClient
```

Comment:
General thing. We call this runner everywhere but it's something inherited from ethers isn't it?  Perhaps we can simply call it client

Reply:
I like the idea, but it would probably introduce breaking changes as well. So I would prefer doing this in a follow-up task